examples/quickstarts/helidon-quickstart-mp/pom.xml

```diff
@@ -42,6 +42,10 @@
             <scope>runtime</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
grpc metrics should not be part of Quickstart MP

Reply:
Sigh. Leftover from an ad hoc test. Fixed.